axum/src/middleware/from_fn.rs

```diff
@@ -71,9 +73,7 @@ use tower_service::Service;
 /// async fn auth(
 ///     // run the `HeaderMap` extractor
 ///     headers: HeaderMap,
-///     // you can also add more extractors here but the last
-///     // extractor must implement `FromRequest` which
-///     // `Request` does
+///     // a `FromRequest` implementation must be the last extractor
```

Comment:
Why change this comment? I don't see how this new version is any clearer. Did the longer version seem confusing?

Reply:
Honestly, I don't think I read the text carefully enough. Reverting this hunk.